units/src/iosMain/swift/Differ/UnitsSource+Diffable.swift

```diff
@@ -40,13 +46,13 @@ extension TableUnitsSourceKt {
             } completion: { _ in
                 let visibleIndexPaths: Array<IndexPath> = tableView.indexPathsForVisibleRows ?? []
                 let cellsToUpdate: Array<IndexPath> = visibleIndexPaths.filter { indexPath in
-                    let newItem: TableUnitItem? = new?.getSafe(indexPath: indexPath)
-                    let noId: Bool = newItem?.itemId == TableUnitItemCompanion().NO_ID
-                    let notInsertOrDelete: Bool = !update.contains(indexPath: indexPath)
-                    return noId || notInsertOrDelete
```

Comment:
No need to reload cells with noId, because they always inserted as new cell by condition in `compareTabelUnitItems`.